packages/stackpress-src/src/plugins/admin/transform/templates/search.ts

```diff
@@ -251,6 +248,9 @@ export default function generate(
   registry: Registry,
   fs: FileSystem
 ) {
+  const searchableTables = ['profile', 'auth', 'application'];
+  const filterableTables = ['profile', 'auth', 'application', 'connection', 'file', 'address', 'session'];
+
```

Comment:
No this is the wrong answer because models are dynamic depending on the idea file.